engine/table/src/main/java/io/deephaven/engine/table/impl/by/LongChunkedAddOnlyMinMaxOperator.java

```diff
@@ -42,7 +49,15 @@ class LongChunkedAddOnlyMinMaxOperator implements IterativeChunkedAggregationOpe
         this.minimum = minimum;
         this.name = name;
         // region resultColumn initialization
-        resultColumn = type == DateTime.class ? new DateTimeArraySource() : new LongArraySource();
+        if (type == DateTime.class) {
```

Comment:
It probably would be nice. As noted offline we're probably in trouble anywhere we lose the ZoneId (say via `ComponentType` usages). I've opened an issue for creating a `ColumnType` that we can pass around instead of `Class<?>`. This is probably a lot better than introducing an optional `ZoneId` to pass through multiple layers.

See #3455.